securesystemslib/key.py

```diff
@@ -95,7 +75,7 @@ def from_dict(cls, key_dict: Dict[str, Any], keyid: str) -> "SSlibKey":
         keyval = key_dict.pop("keyval")
 
         # All fields left in the key_dict are unrecognized.
-        return cls(keyid, keytype, scheme, keyval, key_dict)
+        return cls(keytype, scheme, keyval, keyid, key_dict)
```

Comment:
Why change the argument order?

Reply:
I thought in future `keyid` will get optional, so it's a better position for that.